new_course/1_programming_fundementals/input_and_output.md

```diff
@@ -1,23 +1,73 @@
 <!-- This lesson will cover how to output things to the screen in Ruby and how to get input from the user. -->
 
 ### Introduction
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+When we execute a computer program in Ruby Programming Language, most of the times it is required to either **output** or **input** a piece of information to a computer screen.
+For example, we may need to print (output) the information regarding a value of a number, your first and/or last name, etc. to the terminal. In contrast, we may require from the user to provide (input) the necessary information from the user screen (via using the keyboard) regarding e.g. a number (age), a small text (family’s name) etc.
 
 ### Learning Outcomes
-*Look through these now and then use them to test yourself after doing the assignment*
+* How can I produce an *input* ?
+* How can I produce an *output* ?
+* What is the basic syntax and use of the commands: *print*, *puts*, *gets*
+* What is the difference in input commands (*puts* /*print*) suggested?
 
-what the student is expected to know or be able to do by the end of this lesson
+### Syntax of Basic Commands
+* In order to output the information we want we shall use the **print** command !	
+```ruby
+print "Learning to code is FUN!" #=> will output to our screen the text Learning to code is FUN!
+```
+```ruby
+print "1234" #=> will print to our screen the text 1234
+```
 
-* Learning outcome 1
-* Learning outcome 2
-* Learning outcome 3
+* In order to output the information we want we can also use the **puts** command !	
+```ruby
+puts "Learning to code is cool!!" #=> will output to our screen the text Learning to code is cool!!
+```
+```ruby
+puts "hey I want 2 lemonades" 
+#=> will output to our screen the text hey I want 2 lemonades
+x = "My name is ALEX :)"
+puts x
+   #=> will output to our screen the variable named x in which we have stored the text My name is ALEX :)
+```
 
+* In order to input some information from the user we can use **gets** command!
+```ruby
+y = gets   
+#=> on our command line the program is waiting for user input. E.g. if we type blue sky and press enter
+puts y  
+#=> The output will be blue sky
+#=> will output to our screen the variable named y which has stored the user input (as a text). 
+```
+#### Differences betwenn Basic Commands
 
+Some differences between each command:
 
-### Assignment
-The list of resources the user will go through to learn about the topic of this lesson. Have no more than 5 resources, ideally no more than three.
+1. From the above mentioned examples we can see that for Ruby there is not a difference while declaring a text to have it enclosed between `"` or `'` (single or double quote symbol).
```

Comment:
We can remove this point. We'll be covering this in a lesson that the student will take before this lesson :)